client_cli/src/main.rs

```diff
@@ -243,19 +262,14 @@ mod events {
         }
     }
 
-    pub fn listen(filter: FilterBox, cfg: &Configuration) -> Result<()> {
+    pub fn listen(filter: FilterBox, cfg: &Configuration) -> Result<Box<dyn Serialize>> {
         let iroha_client = Client::new(cfg)?;
-        println!("Listening to events with filter: {filter:?}");
-        for event in iroha_client
+        eprintln!("Listening to events with filter: {filter:?}");
```

Comment:
why are you printing to `stderr`?

Reply:
So that `iroha_client_cli` can be used in scripts, otherwise user would need to somehow separate debug output and the actual invocation results by hand.